src/lib/structures/expr.ml

```diff
@@ -2124,6 +2134,7 @@ module Triggers = struct
              else Var.Map.add v (mk_term (Sy.var Var.underscore) [] ty) sbt
           )t.vars Var.Map.empty
       in
+      (* TODO: remove the useless returned boolean value. *)
```

Comment:
Note: doing this is OK to do in a doc PR in my book, since it is both obviously correct and extremely localized. If the "TODO" is longer than the change itself, might as well do the change.